File: problem001/java/Problem1.java
public class Problem1 {
	public static void main(String[] args) {
		System.out.println(divBy3Or5(1000));
	}

	public static int divBy3Or5(int upto) {
		int val = 0;
        for (int i = 0; i < 1000; ++i) {
            if (i % 3 == 0 || i % 5 == 0) {
                val += i;
            }
        }
        return val;
	}
}

File: problem001/csharp/Problem1.cs
using System.Collections.Generic;
using System.Linq;

public class Problem1 {
	public static void Main() {
        System.Console.WriteLine(
            Enumerable.Range(1, 999)
            .Where(i => ((i % 3 == 0) || (i % 5 == 0)))
            .Sum());
    }
}
